docs/test_instructions_per_dependency_update.md

```diff
@@ -0,0 +1,859 @@
+  works as expected.
+
+</details>
+
+<details>
+  <summary>5. [JP] Notifications Settings [NotificationsSettingsFragment.java]</summary>
+
+- Go to `Notifications` tab.
+- Click on the `Gear` setting button (top-right).
+- Verify that the `Notification Settings` screen is displayed.
+- Click on each of the settings within the `Notification Settings` settings screen and verify that
+  every setting works as expected.
+
+</details>
+
+<details>
+  <summary>6. [JP/WP] Edit Post Screen [EditPostActivity.java]</summary>
+
+ℹ️ Editing a new post uses `PreferenceManager` to `setDefaultValues(...)` for `Account Settings`.
+
+- Go to `Post` screen.
+- Edit a new post, add a few of the main blocks and verify that everything is workings as expected.
+
+</details>
+
+-----
+
+### ExoPlayer [[googleExoPlayerVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="exoplayer"></a>
+
+<details>
+  <summary>1. [JP/WP] Image Preview</summary>
+
+- Go to `Post` screen.
+- Edit a new post, add an `Image` block and then an image into it.
+- Having added an image to the image block, click on the image block and then again on the image within.
+- This will launch `Media Preview` screen and the image should be previewed.
+- Verify that the `Image Preview` is working as expected, that rotating the device works and finally
+  that clicking back navigates you back to the post edit screen.
+
+</details>
+
+<details>
+  <summary>2. [JP/WP] Video Preview</summary>
+
+- Go to `Post` screen.
+- Edit a new post, add an `Video` block but don't add a video into it just yet.
+- Find the `Video` block, click on `ADD VIDEO` and then `Choose from device` to choose a video.
+- From the list of available videos to choose from screen, click on the `Play` button on the center
+  on any video. FYI: If you don't click on the center, this video will get selected for use, but it
+  won't play.
+- This will launch `Media Preview` screen and the video should start playing.
+- Verify that the `Video Preview` is working as expected, that rotating the device works and finally
+  that clicking back navigates you back to the list of available videos to choose from screen.
+
+</details>
+
+-----
+
+### Webkit [[androidxWebkitVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="webkit"></a>
+
+<details>
+    <summary>1. [TODO] TODO</summary>
+
+- TODO
+- TODO
+- TODO
+
+</details>
+
+-----
+
+### Firebase [[firebaseBomVersion](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L1)] <a name="firebase"></a>
+
+<details>
+    <summary>1. [JP/WP] Push Notification on Post Being Published [GCMMessageService.java]</summary>
+
+- Add a new `blog` post.
+- Add any number of blocks.
+- Publish the post.
+- Verify that, on post being published, you get a push notification with the post's title, which,
+  when you click on it, navigates you to the 'Notifications Detail' screen for this post.
```

Comment:
Actually @irfano , the reason I am confident that this is indeed a push notification is because when I do that, and having multiple Jetpack apps installed on my test device, including the production, beta and alpha versions, I get a notification on all 3 apps, including on my iOS test device and the app installed there. 🤔 

Maybe you just need to enable the notifications on your testing device/app by clicking `Fix` on the `Notifications` tab? 🤔

Reply:
Are we talking about the same flow? I am sharing the video of what I am doing and not receiving notification.

[Screen_recording_20231110_163640.webm](https://github.com/wordpress-mobile/WordPress-Android/assets/2471769/efd99639-8561-4173-800d-670040735752)

Reply:
Yes @irfano , as far as I understand we are talking about the same flow... 🤷 

Please check my recording below:

https://github.com/wordpress-mobile/WordPress-Android/assets/9729923/89a9b70e-1420-4379-b2c2-d59c57e26a40

I might be missing something here... 🤔

Reply:
Okay! I have received the notification now with a different account. I believe there are certain criteria on the backend. Even if we use a different test case here, there may still be some criteria for that case. So, let's keep this flow. 👍🏻

Reply:
> Okay! I have received the notification now with a different account.

🎉 

> I believe there are certain criteria on the backend.

Oh, this is a TIL for me that there are some criteria to received push notification! 😮 

> Even if we use a different test case here, there may still be some criteria for that case. So, let's keep this flow. 👍🏻

True, and thanks again for being so thorough @irfano ! 🙇